src/Process/DbDump/DbDump.php

```diff
@@ -110,9 +110,9 @@ public function execute()
                 $this->logger->info('Start creation DB dump...');
 
                 $command = $this->getCommand() . ' | gzip > ' . $dumpFile;
-                $errors = $this->shell->execute('bash -c "set -o pipefail; ' . $command . '"');
+                $process = $this->shell->execute('bash -c "set -o pipefail; ' . $command . '"');
 
-                if ($errors) {
+                if ($process->getExitCode() !== 0) {
```

Comment:
Let's add zero code constant to ShellInterface s `CODE_SUCCESS = 0;`

Reply:
added